### PR TITLE
Update README.rst and pom.xml to 4.0.0

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -345,52 +345,52 @@ The DiskPerformanceApp can be built and packaged using the Apache Maven command:
...
-  $ cdap-cli.sh send stream diskReads \''disk1 1001'\'
-  $ cdap-cli.sh send stream diskReads \''disk1 1001'\'
-  $ cdap-cli.sh send stream diskReads \''disk1 1001'\'
-  $ cdap-cli.sh send stream diskReads \''disk1 1001'\'
-  $ cdap-cli.sh send stream diskReads \''disk1 1001'\'
+  $ cdap cli send stream diskReads \''disk1 1001'\'
+  $ cdap cli send stream diskReads \''disk1 1001'\'
+  $ cdap cli send stream diskReads \''disk1 1001'\'
+  $ cdap cli send stream diskReads \''disk1 1001'\'
+  $ cdap cli send stream diskReads \''disk1 1001'\'
```
Comment (wolf31o2) [inline README.rst]: Why are we using quotes like this? `cdap cli send stream diskReads '"disk1 1001"'` should work. I think we should avoid escaping, where possible.

**Review by wolf31o2 — APPROVED** [review]
One question... since this is an otherwise simple change, not blocking for this PR to be merged

File: README.rst (hunk 1 above)
Comment (awholegunch) [reply README.rst]: I will test these and confirm.

**Comment by awholegunch** [comment]
I have made changes per your comments, and changed the pom.xml. Please re-review.

**Pull request merged**